### PR TITLE
Lucario-Termux

### PR DESCRIPTION
**Pull request by KdB13**
Lucario theme for Termux terminal in Android.

### CODE REVIEW

**Comment by raphamorim** [comment]
Nice! Thanks @KdB13

**Comment by KdB13** [comment]
Thanks for making me contribute... @raphomorim

File: README.md [hunk 1]
```diff
@@ -143,6 +143,13 @@ $ cp Lucario.dvtcolortheme ~/Library/Developer/Xcode/UserData/FontAndColorThemes
 3.  Include the stylesheet in your HTML by including the line ``<link rel="stylesheet" type="text/css" href="lucario.css">``
 4.  The default stylesheet with highlight code blocks rendered with the class **highlight**.
 
+## Termux
+![CSS Example](https://raw.githubusercontent.com/raphamorim/lucario/master/images/termux.png)
+
+1. Make sure that **Termux:Styling** add-on is installed.
+2. Copy the `lucario.colors` file to`~/.termux/colors`.
+2. Run the `colors.sh` script to apply.
```
Comment (folixg) [inline README.md]: @KdB13 I noticed that here a `colors.sh` script is referenced, but this script is not part of the repository. Where can the user obtain it?

Comment (KdB13) [reply README.md:151]: Oh! I forgot to add that `colors.sh` is only available if the **oh-my-zsh** is installed. I wrote the wrong steps. To apply the color with **Termux:Styling**, we need to replace that file with `~/.termux/colors.properties` while preserving the name `colors.properties`. 

Thanks for bringing this up.

Comment (KdB13) [reply README.md:151]: I've changed the documentation in #66.

**Pull request merged**